src/pages/mypage/components/button/smallButton/SmallButton.tsx

```
@@ -0,0 +1,19 @@
+interface SmallButtonProps {
+  onClick?: () => void;
+  children?: React.ReactNode;
+}
+
+const SmallButton = ({
+  onClick,
+  children = '이미지 더보기',
+}: SmallButtonProps) => {
+  return (
+    <button className={styles.smallButton} onClick={onClick}>
+      {children}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)